src/components/FormAlert/index.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import {
+  themeColor,
+  Heading,
+  ascDefaultTheme,
+  themeSpacing,
+} from '@datapunt/asc-ui';
+
+const Div = styled.div`
+  background-color: ${themeColor('tint', 'level2')};
+  border-left: ${themeSpacing(0.5)} solid;
+  margin-bottom: ${themeSpacing(6)};
+  padding: ${themeSpacing(4)};
+  border-color: ${({ isNotification }) =>
+    isNotification ? themeColor('support', 'valid') : themeColor('secondary')};
+
+  @media (min-width: ${ascDefaultTheme.layouts.small.max}px) {
+    font-size: 18px;
+    padding-left: ${themeSpacing(5)};
+    margin-bottom: ${themeSpacing(8)};
+  }
+`;
+
+const Title = styled(Heading).attrs({
+  $as: 'h4',
+})`
+  font-family: Avenir Next LT W01 Demi, arial, sans-serif;
+  color: ${({ isNotification }) =>
+    isNotification ? themeColor('support', 'valid') : themeColor('secondary')};
+
+  ${({ hasMessage }) =>
+    !hasMessage &&
+    css`
+      margin-bottom: 0;
+    `}
+`;
+
+/**
+ * Component that will display a title and, optionally, a message block. Should be used
+ * for generic feedback that is not related to individual fields.
+ */
+const FormAlert = ({ className, message, title, isNotification }) => (
+  <Div className={className} isNotification={isNotification}>
```

Comment:
this should be called Alert I think there is also FormError component in DS.

it should have 4 variants:
[https://designsystem.amsterdam.nl/7awj1hc9f/p/901ede ](url)

I don't see those

Reply:
What should be called `Alert`? The Component that is the default export or the `<Div>` component on line 45?

This particular component is not meant as a generic notification component, but is specific to the context of a form. That is why this `FormAlert` component doesn't have the four different states that are in the design system. It is not necessary for the current scope. If needed, we can change the component at a later date, but for now, I'm not going to bother investing time in functionality that we don't yet need.

Reply:
IMHO, `Notification` is too generic a name for a component that is meant to show feedback related to actions performed in forms.